### PR TITLE
Apoka Pack Confrontation Cycle - 03

### PR DESCRIPTION
**Pull request by Frohlemageblanc**
Adding Apoka Pack - Cycle 3

### CODE REVIEW

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+        </card>
+        <card id="ef1c60d4-cef7-11e7-abc4-cec278b6b50a" name="Starmist Raiment">
+            <property name="Faction"        value="Eldar"/>
+            <property name="Type"           value="Attachment"/>
+            <property name="Loyalty"        value="Squad (1)"/>
+            <property name="Traits"         value="Wargear. Armor."/>
+            <property name="Text"           value="Attach to a Harlequin unit you control.
+Interrupt: When damage is assigned to a Harlequin unit you control at this planet, exhaust this attachment to reassign 1 of that damage to another non-warlord unit at this planet."/>
+            <property name="Keywords" value=""/>
+            <property name="Unique"         value="1"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Should be 'yes' for Unique I think

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 2]
```diff
@@ -0,0 +1,587 @@
...
+            <property name="Squad"          value="39"/>
+        </card>
+        <card id="ef1c649e-cef7-11e7-abc4-cec278b6b50a" name="Convent Prioris Advisor">
+            <property name="Faction"        value="Astra Militarum"/>
+            <property name="Type"           value="Support"/>
+            <property name="Loyalty"        value="Loyal"/>
+            <property name="Traits"         value="Advisor. Ecclesiarchy."/>
+            <property name="Text"           value="Reaction: After a Vostroya unit moves to a planet, place 1 F on it. Then you may sacrifice this support to place 1 F on that unit and Rally 6 a support, add it to your hand."/>
+            <property name="Keywords" value=""/>
+            <property name="Unique"         value="Yes"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Think 'Yes' needs to be lowercase 'y' for the proxy generation

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+            <property name="Squad"          value=""/>
+        </card>
+        <card id="ef1c662e-cef7-11e7-abc4-cec278b6b50a" name="Command Bravestorm">
+            <property name="Faction"        value="Tau"/>
+            <property name="Type"           value="Army"/>
+            <property name="Loyalty"        value="Yes"/>
+            <property name="Traits"         value="Soldier. Pilot. Hero."/>
+            <property name="Text"           value="Reaction: After you attach a non-Drone attachment to a unit at this planet, draw a card. (Limit once per phase.)"/>
+            <property name="Keywords" value=""/>
+            <property name="Unique"         value="Yes"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: yes

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 4]
```diff
@@ -0,0 +1,587 @@
...
+            <property name="CardBonus"      value=""/>
+            <property name="ErrataText"     value=""/>
+            <property name="CardNumber"     value="16"/>
+            <property name="Copies"         value=""/>
+            <property name="Squad"          value=""/>
+        </card>
+        <card id="ef1c662e-cef7-11e7-abc4-cec278b6b50a" name="Command Bravestorm">
+            <property name="Faction"        value="Tau"/>
+            <property name="Type"           value="Army"/>
+            <property name="Loyalty"        value="Yes"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Think the value for Loyalty is 'Loyal' or blank

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 5]
```diff
@@ -0,0 +1,587 @@
...
+            <property name="CardBonus"      value=""/>
+            <property name="ErrataText"     value=""/>
+            <property name="CardNumber"     value="17"/>
+            <property name="Copies"         value=""/>
+            <property name="Squad"          value=""/>
+        </card>
+        <card id="ef1c6886-cef7-11e7-abc4-cec278b6b50a" name="Phantasmatic Masque">
+            <property name="Faction"        value="Eldar"/>
+            <property name="Type"           value="Army"/>
+            <property name="Loyalty"        value="Yes"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Loyal

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 6]
```diff
@@ -0,0 +1,587 @@
...
+            <property name="CardBonus"      value=""/>
+            <property name="ErrataText"     value=""/>
+            <property name="CardNumber"     value="18"/>
+            <property name="Copies"         value=""/>
+            <property name="Squad"          value=""/>
+        </card>
+        <card id="ef1c694e-cef7-11e7-abc4-cec278b6b50a" name="Connoisseur of Terror">
+            <property name="Faction"        value="Dark Eldar"/>
+            <property name="Type"           value="Army"/>
+            <property name="Loyalty"        value="Yes"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Loyal

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 7]
```diff
@@ -0,0 +1,587 @@
...
+            <property name="CardBonus"      value=""/>
+            <property name="ErrataText"     value=""/>
+            <property name="CardNumber"     value="19"/>
+            <property name="Copies"         value=""/>
+            <property name="Squad"          value=""/>
+        </card>
+        <card id="ef1c6a16-cef7-11e7-abc4-cec278b6b50a" name="Advocator of Blood">
+            <property name="Faction"        value="Chaos"/>
+            <property name="Type"           value="Army"/>
+            <property name="Loyalty"        value="Yes"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Loyal

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml [hunk 8]
```diff
@@ -0,0 +1,587 @@
...
+            <property name="CardBonus"      value=""/>
+            <property name="ErrataText"     value=""/>
+            <property name="CardNumber"     value="20"/>
+            <property name="Copies"         value=""/>
+            <property name="Squad"          value=""/>
+        </card>
+        <card id="ef1c6ade-cef7-11e7-abc4-cec278b6b50a" name="Ireful Vanguard">
+            <property name="Faction"        value="Chaos"/>
+            <property name="Type"           value="Army"/>
+            <property name="Loyalty"        value="Yes"/>
```
Comment (blakewedwards) [inline af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Loyal

File: af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml (hunk 1 above)
Comment (Frohlemageblanc) [reply af04f855-58c4-4db3-a191-45fe33381679/Sets/ef1c72d6-cef7-11e7-abc4-cec278b6b50a/set.xml]: Well I did not have any invalid card image with the latest commit but did not try to check the deck validation. Switching for new values then.

**Pull request merged**